### ISSUE TITLE
Stopped working (as of FF 20.0.1)

### ISSUE THREAD

**Issue by Dexterprog** [nedwidek/firefox-gmail-manager-ng#32]
Today, the addon has stopped working for me. It's the first time I've had an issue since it was released, perhaps something changed in gmail?

**Comment by tester2011** [comment]
Yep, same here. The issue seems to be that privacy mode (in FF 20.+) causes some sort of cookies issue, which gives an error when signing into gmail.
